CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="4.6.0"></a>
+
+## [4.6.0] - 2024-07-10
```

Comment:
could we add more info about new methods that are being added?

Reply:
Right will update the changelog entry to include the other items being added as a part of this change.